src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1178,6 +1178,7 @@ public static partial class Path
         public static string GetTempPath() { throw null; }
         public static bool HasExtension(string path) { throw null; }
         public static bool IsPathRooted(string path) { throw null; }
+        public static bool IsPathFullyQualified(string path) { throw null; }
```

Comment:
the title of the PR suggests that you are only adding tests but here you are adding a public API. Intentional?